autoscheduler/frontend/src/types/Availability.ts

```diff
@@ -0,0 +1,38 @@
+export enum AvailabilityType {
+  NONE, BUSY
+}
+
+export default interface Availability {
+  dayOfWeek: number;
+  startTimeHours: number;
+  startTimeMinutes: number;
+  endTimeHours: number;
+  endTimeMinutes: number;
+  available: AvailabilityType;
+// eslint-disable-next-line semi
```

Comment:
Put this in `eslintrc` rules instead:
```
  "semi": "off",
  "@typescript-eslint/semi": ["error"],
```
I got the rules from [this](https://github.com/typescript-eslint/typescript-eslint/issues/123) page

Reply:
Oh, okay, so replace the normal rule with the TS one. Thanks!